doc/source/whatsnew/v2.2.0.rst

```diff
@@ -77,7 +77,7 @@ Other enhancements
 - :func:`read_csv` now supports ``on_bad_lines`` parameter with ``engine="pyarrow"``. (:issue:`54480`)
 - :meth:`ExtensionArray._explode` interface method added to allow extension type implementations of the ``explode`` method (:issue:`54833`)
 - DataFrame.apply now allows the usage of numba (via ``engine="numba"``) to JIT compile the passed function, allowing for potential speedups (:issue:`54666`)
-- Implement masked algorithms for :meth:`Series.value_counts` (:issue:`54984`)
+- Implement masked algorithms for :meth:`Series.value_counts` and :meth:`Series.mode` (:issue:`54984`, :issue:`55340`)
```

Comment:
users mainly need to know this improves performance?

Reply:
moved and reworded